packages/client-sdk-nodejs/src/internal/data-client.ts

```diff
@@ -158,6 +159,69 @@ export class DataClient implements IDataClient {
       this.configuration.getMiddlewares()
     );
   }
+  public connect(): Promise<void> {
+    const deadline = new Date();
+    deadline.setSeconds(deadline.getSeconds() + 10);
```

Comment:
Should this be more like 1 second since we know on average we took ~200ms?

Reply:
I don't know what it would be in other environments since the customer was seeing 5 second latencies. I can make it configurable if that works better.

Reply:
It has been made configurable.